src/main/java/org/folio/service/transactions/batch/BatchEncumbranceService.java

```diff
@@ -71,12 +71,14 @@ private void updateBudget(Budget budget, CurrencyUnit currency, Transaction encu
       double newAmount = subtractMoney(encumbrance.getEncumbrance().getInitialAmountEncumbered(),
         existingEncumbrance.getEncumbrance().getAmountAwaitingPayment(), currency);
       newAmount = subtractMoney(newAmount, existingEncumbrance.getEncumbrance().getAmountExpended(), currency);
+      newAmount = subtractMoney(newAmount, existingEncumbrance.getEncumbrance().getAmountCredited(), currency);
       encumbrance.setAmount(newAmount);
       newEncumbered = sumMoney(currency, newEncumbered, newAmount);
     } else if (isTransitionFromReleasedToUnreleased(encumbrance, existingEncumbrance)) {
       double newAmount = subtractMoney(encumbrance.getEncumbrance().getInitialAmountEncumbered(),
         encumbrance.getEncumbrance().getAmountAwaitingPayment(), currency);
       newAmount = subtractMoney(newAmount, encumbrance.getEncumbrance().getAmountExpended(), currency);
+      newAmount = subtractMoney(newAmount, encumbrance.getEncumbrance().getAmountCredited(), currency);
```

Comment:
shouldn't that be added instead, since expended is subtracted ?

Reply:
https://folio-org.atlassian.net/browse/MODFISTO-481?focusedCommentId=215883, I did base on your comment. I thought both should be subtracted. If you are confident about that, I will change that

Reply:
My comment suggested to replace `amountExpended` by `amountExpended - amountCredited`. So `newAmount = newAmount - amountExpended` should become `newAmount = newAmount - (amountExpended - amountCredited)` which is equivalent to `newAmount = newAmount - amountExpended + amountCredited`.

Reply:
thanks, for explanation, fixed